ir/constant/const_float.go

```diff
@@ -194,9 +194,6 @@ func NewFloatFromString(typ *types.FloatType, s string) (*Float, error) {
 				panic(fmt.Errorf("support for hexadecimal floating-point literal %q of kind %v not yet implemented", s, typ.Kind))
 			}
 		}
-		log.Printf("constant.NewFloatFromString(%q): not yet implemented", s)
```

Comment:
Yay, happy to see full support for hexadecimal floating-point literals finally being implemented. Great job @dannypsnl!

Reply:
https://github.com/llir/llvm/pull/123/files#diff-296597e1a91f88998cc9640e7b83fcf5R11-R26

tests are there I thought?

Reply:
Ah, I missed those.